configure.ml

```diff
@@ -343,7 +344,9 @@ let args_options = Arg.align [
   "-profile", Arg.Set Prefs.profile,
     " Add profiling information in the Coq executables";
   "-annotate", Arg.Set Prefs.annotate,
-    " Dumps ml annotation files while compiling Coq";
+    " Dumps ml text annotation files while compiling Coq";
+  "-bin-annotate", Arg.Set Prefs.bin_annotate,
+    " Dumps ml binary annotation files while compiling Coq";
```

Comment:
Please include in the description some hints on which option is useful for what (Tuareg users vs Merlin users).